android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/MetadataResolver.java

```diff
@@ -236,6 +249,17 @@ public void onErrorResponse(VolleyError volleyError) {
     );
   }
 
+  public static boolean checkIfMetadataContainsSortingScores(Collection<UrlMetadata> urlMetadataCollection) {
```

Comment:
I think that's a fair concern.

Keep in mind that it's a transitional state of the server. Eventually, the server will always send scores. Therefore, I think it would be overkill to create a new webservice to know the server capabilities.

What we can do is making the server transition smooth for the client.

I think we can proceed as follow:
- We can check in that temporary code on the client
- Move the server implementation to proxy sorting
- Then, clean up that temporary code and remove the client-side sorting. We won't have to check whether the score is available or not.

What do you think?